src/modules/pos/validator/components/ValidatorsMonitorView/Validators.css

```diff
@@ -322,3 +322,61 @@ header {
   border-radius: 30px;
   min-width: 250px !important;
 }
+
+.bannerSwiper {
```

Comment:
Style isn't used in component and is a duplicate of the similar style in SwippableBanner component and should therefore be removed. The same should be done for `BlockchainApplications` component

Reply:
Fixed